src/main/java/core/basesyntax/service/operations/SupplyOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.model.FruitTransaction;
+
+public class SupplyOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction) {
+        storageFruits.put(transaction.getFruit(), storageFruits
```

Comment:
It appears that `storageFruits` is being used without being defined in this class. You should define a storage instance within this class or pass it as a parameter to the `handle` method. Also, ensure that `storageFruits` is a part of the `db` package as per the checklist.